index.js

```diff
@@ -190,6 +191,7 @@ const isEmptyMapOrSet = x => (is.map(x) || is.set(x)) && x.size === 0;
 
 is.empty = x => !x || isEmptyStringOrArray(x) || isEmptyObject(x) || isEmptyMapOrSet(x);
 is.emptyOrWhitespace = x => is.empty(x) || isWhiteSpaceString(x);
+is.emoji = x => is.string(x) && emojiRegex.test(x);
```

Comment:
The name implies we are checking for a single emoji, but this currently returns true for any string that _contains_ an emoji:

```js
emojiRegex.test('this is a test 🙏')
// => true
```

Should we consider a different name? Or behavior? Something along the lines of `is.stringWithEmoji` maybe.

Reply:
`is.stringWithEmoji` could work 🤔 let's see what Sindre has to say. 👍